SU-SWS/stanford_basic
# D8CORE-639

# READY FOR REVIEW

# Summary
- This PR depends on https://github.com/SU-SWS/stanford_basic/pull/89

# Review By (Date)
- October 18th

# Urgency
- Low

# Steps to Test

1. Check out this branch
2. Compile the sass `npm run publish`
3. Run a fresh install `blt drupal:install`
4. Review the default components (paragraphs) for the modular margin on the bottom of each paragraph

# Affected Projects or Products
- All paragraphs

# Associated Issues and/or People
- D8CORE-639
- Make @kerri-augenstein happy.
- @pookmish @boznik FYI

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
